tests/integration/Dockerfile.girder

```diff
@@ -0,0 +1,29 @@
+FROM girder/girder:latest
+MAINTAINER Christopher Kotfila <chris.kotfila@kitware.com>
+
+VOLUME /girder/plugins/integration_test_endpoints
+
+RUN sed -i 's|^uri = "mongodb://localhost:27017/girder"$|uri = "mongodb://mongo:27017/girder"|' /girder/girder/conf/girder.local.cfg
+RUN sed -i 's|^server.socket_port = 8080$|server.socket_port = 8989|' /girder/girder/conf/girder.local.cfg
```

Comment:
Should these just be moved to the entrypoint since they're both flags `python -m girder` takes? I think that's less brittle than sed.